package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "build": "tsc",
     "prepublishOnly": "npm run build",
-    "test": "ts-mocha -p tsconfig.json src/__test__/**/*.test.ts",
+    "test": "mocha --require ts-node/register --require source-map-support/register --recursive src/__test__/**/*.test.ts",
```

Comment:
This gives us test coverage support. Not really needed, but was useful when setting up tests. The test:coverage actually runs the test with coverage, so you can view the report etc in your ide.